### PR TITLE
DEV: travis file added

### PR DESCRIPTION
**Pull request by licode**

### CODE REVIEW

**Comment by licode** [comment]
Can anyone allow me to access to settings? So I can add travisCI service.

**Comment by licode** [comment]
OK, Travis is working basically. Now I need to figure out errors from test functions.

**Comment by licode** [comment]
Done. @tacaswell ?

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,5 @@
 language: python
 
-services:
```
Comment (ghost) [inline .travis.yml]: We have overhauled the travis file in mdstore as a result of adding authentication to mongo shells. Please check what I have done [here](https://github.com/arkilic/metadatastore/blob/authenticate_mongoserv/.travis.yml)

Comment (tacaswell) [reply .travis.yml]: I do not see how that is relevant to this repo

Comment (ghost) [reply .travis.yml]: I just looked at the travis file (not the repo name), which contains a mongo server. Since this is portable_mds, there is no need for travis to build a mongo server.

File: portable_mds/tests/test_mds.py [hunk 2]
```diff
@@ -2,25 +2,25 @@
...
-    test_event_descriptor_insertion,
-    # test_fail_runstart,
-    test_find_run_start,
-    test_find_run_stop,
-    test_insert_run_start,
-    # test_no_evdesc,
-    test_pickle,
-    test_run_stop_by_run_start,
-    test_run_stop_insertion)
+# from metadatastore.test.test_mds import (
```
Comment (tacaswell) [inline portable_mds/tests/test_mds.py]: Why did you remove all of the tests?

File: .travis.yml (hunk 1 above)
Comment (licode) [reply .travis.yml]: Yes. There is no mongo server here.

Comment (ghost) [reply .travis.yml]: 👦  - ☕  = 😴

File: portable_mds/tests/test_mds.py (hunk 2 above)
Comment (licode) [reply portable_mds/tests/test_mds.py]: I thought we already ran those tests in metadatastore. 

It looks like we redefine mds_all again in pytest.fixture, so we need to run every test again here. I will get them back.

Comment (ghost) [reply portable_mds/tests/test_mds.py]: The goal of the fixture here is to make sure portable_mds passes all the metadatastore tests w/o having to rewrite them. mdservice can even use portable_mds as its backend (I tested this, it does 👏 ) .

**Comment by tacaswell** [comment]
These failures are likely due to https://github.com/NSLS-II/metadatastore/pull/239 

Can you port these changes to portable_mds?

**Comment by licode** [comment]
Sure, I will give it a try.

**Comment by licode** [comment]
This should be in good shape.
